pyomo/contrib/mindtpy/config_options.py

```diff
@@ -123,48 +83,16 @@ def _get_GDPopt_config():
         description="Stream output to terminal.",
         domain=bool
     ))
-    CONFIG.declare("solver_tee", ConfigValue(
-        default=False,
-        description="Stream the output of mip solver and nlp solver to terminal.",
-        domain=bool
-    ))
     CONFIG.declare("logger", ConfigValue(
         default='pyomo.contrib.mindtpy',
         description="The logger object or name to use for reporting.",
         domain=a_logger
     ))
-    CONFIG.declare("small_dual_tolerance", ConfigValue(
-        default=1E-8,
-        description="When generating cuts, small duals multiplied "
-                    "by expressions can cause problems. Exclude all duals "
-                    "smaller in absolute value than the following."
-    ))
-    CONFIG.declare("integer_tolerance", ConfigValue(
-        default=1E-5,
-        description="Tolerance on integral values."
-    ))
-    CONFIG.declare("constraint_tolerance", ConfigValue(
-        default=1E-6,
-        description="Tolerance on constraint satisfaction."
-    ))
-    CONFIG.declare("variable_tolerance", ConfigValue(
-        default=1E-8,
-        description="Tolerance on variable bounds."
-    ))
-    CONFIG.declare("zero_tolerance", ConfigValue(
-        default=1E-8,
-        description="Tolerance on variable equal to zero."
-    ))
     CONFIG.declare("initial_feas", ConfigValue(
```

Comment:
Is this the feasibility pump? If so, leave it `False` by default and change the name to a more complete `initial_feaspump`.

Reply:
I checked the code. This is an ancient option created before the feasibility pump branch. It's somehow useless, so I deleted it.